services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/client_api.py

```diff
@@ -218,7 +218,77 @@ async def service_pull_input_ports(
             )
             if response.status_code != status.HTTP_200_OK:
                 message = (
-                    f"ERROR while restoring service state: "
+                    f"ERROR while pulling input ports: "
+                    f"status={response.status_code}, body={response.text}"
+                )
+                logging.warning(message)
+                raise DynamicSchedulerException(message)
+            return int(response.text)
+        except httpx.HTTPError as e:
+            log_httpx_http_error(url, "POST", traceback.format_exc())
+            raise e
+
+    async def service_disable_dir_watcher(self, dynamic_sidecar_endpoint: str) -> None:
+        url = get_url(dynamic_sidecar_endpoint, "/v1/containers/directory-watcher")
+        try:
+            response = await self._client.patch(url, json=dict(is_enabled=False))
+            if response.status_code != status.HTTP_204_NO_CONTENT:
+                message = (
+                    f"ERROR while disabling directory watcher: "
```

Comment:
would it not be simpler and easier to wrape the message in the exception (which you do) and then display this where the exception gets catched? whay is the usefulness of having everywhere such code?